### DISCUSSION TITLE
ImGui on macOS 14 w/ OpenGL 3.3 crashes

### DISCUSSION THREAD

**Discussion by bsdooby** [ocornut/imgui#7644] [New Users Build/Link/Run issues ONLY!]
I cannot use ImGui w/ my app (based on OpenGL 3.3, macOS 14 Sonoma); I use GLFW as the extension loader; as soon
as I activate and initialize ImGui, the app crashes; the same code base on Linux w/ ImGui activated works as intended.

**Comment by PathogenDavid** [comment]
Does the official demo crash?

Either way you'll need to provide more details, like a stack trace of the crash

**Comment by bsdooby** [comment]
Will do more testing, and report back. Stay tuned.

**Comment by bsdooby** [comment]
After updating to the latest, and enabling this section (was not needed w/
earlier versions on macOS (?)):


//#if defined(__linux__) || defined(_WIN32)

glewExperimental = true;

glewInit();

//#endif


And setting "#version 330" for the GLSL version string.

it seems to work now.


Not exactly sure about the issue.

**Reply by ocornut** [reply]
Please answer the asked questions, that would be useful. Clarify where/how it crashes.

**Comment by bsdooby** [comment]
As stated, it would crash in my renderer without using/initializing GLFW.
The ImGui provided glfw opengl3 example code works as advertised.

So, not an issue w/ ImGui, but with my code in combination w/ GLFW on macOS.

**Reply by ocornut** [reply]
But you are posting here implying requesting help about your crash and not specifying how it crashes. What can we do?

**Comment by bsdooby** [comment]
All is good now, and I provided the "solution": it is not an ImGui issue.
So thank you for your support, and have a great day.
